### PR TITLE
Run service as separate user

### PR DESCRIPTION
**Pull request by nxshock**

### CODE REVIEW

**Comment by Revertron** [comment]
Can you also add user creation here https://github.com/Revertron/Alfis/blob/master/contrib/deb/generate.sh ?
Near adduser.

**Comment by nxshock** [comment]
I'm not sure how to do it for Debian based distros. This changes needs to be tested.

**Comment by Revertron** [comment]
> 
> 
> I'm not sure how to do it for Debian based distros. This changes needs to be tested.

I meant here: https://github.com/Revertron/Alfis/blob/master/contrib/deb/generate.sh#L76
Just add some `adduser alfis` maybe. Not in files in /usr...

File: contrib/deb/generate.sh [hunk 1]
```diff
@@ -77,6 +72,8 @@ EOF
 cat > /tmp/$PKGNAME/debian/postinst << EOF
 #!/bin/sh
 
+id -u somename &>/dev/null || useradd --system alfis || echo "Failed to create user 'alfis' - please create it manually and reinstall"
```
Comment (Revertron) [inline contrib/deb/generate.sh]: `somename`?

**Review by Revertron — APPROVED** [review]

File: contrib/deb/generate.sh (hunk 1 above)
Comment (nxshock) [reply contrib/deb/generate.sh]: Already fixed in ef9c265

**Pull request merged**